e3sm_diags/derivations/derivations.py

```diff
@@ -665,417 +680,184 @@
         ("FISCCP1_COSP",): cosp_histogram_standardize,
         ("CLISCCP",): cosp_histogram_standardize,
     },
-    "ICEFRAC": OrderedDict(
```

Comment:
I replaced unnecessary OrderedDict with a regular dictionary, but many still remain. All OrderedDicts should eventually be replaced in #716 because they are hard to read and work with.